st2common/st2common/models/api/rule.py

```diff
@@ -162,7 +163,7 @@ def to_model(cls, rule):
 
         trigger = reference.get_str_resource_ref_from_model(trigger_db)
         criteria = dict(getattr(rule, 'criteria', {}))
-        pack = str(rule.pack)
+        pack = getattr(rule, 'pack', DEFAULT_PACK_NAME)
```

Comment:
It's also worth clarifying, that as far as I know, this only applies to the rules which were created using the API before we added `pack` field to the `Rule` model.

When we added `pack` field, we also added a change which sets `pack` to a default value if the value is not provided in the API request payload.